components/web/nginx/Makefile

```diff
@@ -13,63 +13,63 @@
 #
 
 BUILD_BITS= 64
-# Alas nginx ignores the openssl settings
-OPENSSL_VERSION= 3.1
-USE_PARALLEL_BUILD= yes
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME=		nginx
-COMPONENT_VERSION=	1.25.4
-COMPONENT_SUMMARY=	Nginx Webserver
-COMPONENT_PROJECT_URL=	https://nginx.net/
-COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_HASH=	sha256:760729901acbaa517996e681ee6ea259032985e37c2768beef80df3a877deed9
-COMPONENT_ARCHIVE_URL=	https://nginx.org/download/$(COMPONENT_ARCHIVE)
-COMPONENT_SIG_URL=	$(COMPONENT_ARCHIVE_URL).asc
+COMPONENT_NAME= nginx
+COMPONENT_VERSION= 1.25.4
+COMPONENT_SUMMARY= Nginx Webserver
+COMPONENT_PROJECT_URL= https://nginx.org/
+COMPONENT_SRC= $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:760729901acbaa517996e681ee6ea259032985e37c2768beef80df3a877deed9
+COMPONENT_ARCHIVE_URL= https://nginx.org/download/$(COMPONENT_ARCHIVE)
+COMPONENT_SIG_URL= $(COMPONENT_ARCHIVE_URL).asc
 COMPONENT_FMRI=	web/server/nginx
-COMPONENT_CLASSIFICATION=	Web Services/Application and Web Servers
-COMPONENT_LICENSE=	BSD
+COMPONENT_CLASSIFICATION= Web Services/Application and Web Servers
+COMPONENT_LICENSE= BSD
 COMPONENT_LICENSE_FILE=	LICENSE
+COMPONENT_REVISION= 1
```

Comment:
Please move this right below `COMPONENT_VERSION` so it is not forgotten on the next version update.  Thank you.